src/frontend/typer/oppgave.ts

```diff
@@ -108,6 +110,18 @@ export const gjelderFilter: INøkkelPar = {
     ab0058: { id: 'ab0058', navn: 'EØS' },
 };
 
+export enum BehandlingstypeFilter {
+    ALLE = 'ALLE',
+    ae0106 = 'ae0106',
+    ae0161 = 'ae0161',
+}
+
+export const behandlingstypeFilter: INøkkelPar = {
+    ALLE: { id: 'ALLE', navn: 'Alle' },
+    ae0106: { id: 'ae0106', navn: 'Utland' },
+    ae0161: { id: 'ae0161', navn: 'Tilbakekreving' },
+};
```

Comment:
Bare pirk, så du trenger ikke gjøre noe med det hvis det er mye jobb: 
Tror vi har prøvd å gå litt bort fra INøkkelPar og heller lage records (feks `Record<VedtakBegrunnelseType, string>`), men mulig det er kronglete å bruke det her, siden resten av filterne sikkert er definert som INøkkelPar..

Reply:
Tenkte det samme selv, men tror det er slikt som kan gjøres i en generell forbedring av oppgavebenken :)